### PR TITLE
Minor edits to the book.

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,4 +27,4 @@ You mean besides incredibly good karma? Here are some very likely outcomes:
 - You’ll meet interesting new people outside your usual developer group.
 - You’ll probably learn something new by answering questions you never thought of asking yourself.
 - Eternal gratitude from your group of attendees and the Django Girls community.
-- You’ll help break down the stereotype that women are not interested in technology and programming and thus help your sisters, female friends and relatives, (future) daughters have more choices in their future. And you can add all that on your CV!
+- You’ll help break down the stereotype that women are not interested in technology and programming and thus help your sisters, female friends and relatives, and (future) daughters have more choices in their future. And you can add all that on your CV!
```
Comment (aniav) [inline README.md]: I believie this "and" here isn't needed.

Comment (timgraham) [inline README.md]: When you have a couple things in a list, usually you say "A, B, and C" (unless I'm misreading this)

Comment (aniav) [inline README.md]: Yes, but here you have "and" two times in one sentence. The proper one should be:
```
You’ll help break down the stereotype that women are not interested in technology and programming and thus help your sisters, female friends, relatives and (future) daughters have more choices in their future.```

I'm also not sure that you should place a comma before the and in the sentence.

Comment (timgraham) [inline README.md]: Okay, that version is fine. Regarding the comma before "and", see http://en.wikipedia.org/wiki/Serial_comma. It seems to be a mostly American thing...

**Comment by aniav** [comment]
I will merge your commits with some small changes, so we will be able to close this. It already hangs a long time here :)

**Comment by timgraham** [comment]
Thanks Anna.

**Comment by aniav** [comment]
And a great thank you for all your effort! :+1:

**Pull request merged**